src/org/montclairrobotics/sprocket/drive/DriveModule.java

```diff
@@ -1,47 +1,53 @@
 package org.montclairrobotics.sprocket.drive;
 
-import edu.wpi.first.wpilibj.SpeedController;
 import org.montclairrobotics.sprocket.geometry.Angle;
-import org.montclairrobotics.sprocket.geometry.Inch;
+import org.montclairrobotics.sprocket.geometry.Radians;
 import org.montclairrobotics.sprocket.geometry.Vector;
-import org.montclairrobotics.sprocket.utils.PID;
-
-public class DriveModule extends Motor {
-
-    private Angle forceAngle;
-    private Vector offset;
-    private Inch maxSpeed;
-
-    public DriveModule(SpeedController motor,
-                       Angle forceAngle, Vector offset,
-                       SEncoder enc, PID pid, Inch maxSpeed,
-                       boolean invert) {
-        super(motor, enc, pid, invert);
-        this.forceAngle = forceAngle;
-        this.offset = offset;
-        this.maxSpeed = maxSpeed;
-    }
-
-    public DriveModule(SpeedController motor,
-                       Angle forceAngle, Vector offset,
-                       SEncoder enc, PID pid, Inch maxSpeed) {
-        this(motor, forceAngle, offset, enc, pid, maxSpeed, false);
-    }
-
-    public DriveModule(SpeedController motor,
-                       Angle forceAngle, Vector offset) {
-        this(motor, forceAngle, offset, null, null, null, false);
-    }
-
-    public Angle getForceAngle() {
-        return forceAngle;
-    }
-
-    public Inch getMaxSpeed() {
-        return maxSpeed;
-    }
-
-    public Vector getOffset() {
-        return offset;
-    }
+import org.montclairrobotics.sprocket.loop.Priority;
+import org.montclairrobotics.sprocket.loop.Updatable;
+import org.montclairrobotics.sprocket.loop.Updater;
+import org.montclairrobotics.sprocket.pipeline.Pipeline;
+
+public class DriveModule implements Updatable{
```

Comment:
Should all this math go in the drive module class? I think it would be better to separate the power mapping stuff into a power mapper and leave the drive module class for handling PIDs and controlling motor powers.